crates/experimental-api/Cargo.toml

```diff
@@ -18,10 +18,15 @@ tracing.workspace = true
 thiserror.workspace = true
 
 starknet-crypto = "0.6.1"
-blockifier = { git = "https://github.com/starkware-libs/blockifier", tag = "v0.5.0-rc.2" }
-starknet_api = "0.8.0"
+blockifier = { git = "https://github.com/starkware-libs/blockifier", tag = "v0.5.0" }
+starknet_api = "0.10.0"
 cairo-vm = "0.9.2"
 hex = "0.4.3"
 
 base64 = "0.22.0"
 flate2 = "1.0.28"
+cairo-lang-starknet-classes = "2.6.3"
+ureq = { version = "2.9.6", features = ["json"] }
```

Comment:
Why introduce a new HTTP client when the rest of the project relies on `reqwest`?

Reply:
https://github.com/eigerco/beerus/issues/626